### ISSUE TITLE
identified-biosynthetic-reads.fasta is empty

### ISSUE THREAD

**Issue by aimirza** [donia-lab/MetaBGC#16]
I ran the command on one of my samples to test. The file named `identified-biosynthetic-reads.fasta` is empty. The file `CombinedHmmSearch.txt` is not empty and has many hits. However, the file `CombinedReadIds.txt` is empty and the file `spHMM-filtered-results.txt` has no results but it does have column headers. The directory `fasta_seq_result/` is empty. The directory `hmm_identify_search/` contains:

```
LON-00010-PT-02_S5.__0_30.tbl     LON-00010-PT-02_S5.__170_200.txt  LON-00010-PT-02_S5.__210_240.tbl  LON-00010-PT-02_S5.__70_100.txt
LON-00010-PT-02_S5.__0_30.txt     LON-00010-PT-02_S5.__180_210.tbl  LON-00010-PT-02_S5.__210_240.txt  LON-00010-PT-02_S5.__80_110.tbl
LON-00010-PT-02_S5.__130_160.tbl  LON-00010-PT-02_S5.__180_210.txt  LON-00010-PT-02_S5.__220_250.tbl  LON-00010-PT-02_S5.__80_110.txt
LON-00010-PT-02_S5.__130_160.txt  LON-00010-PT-02_S5.__190_220.tbl  LON-00010-PT-02_S5.__220_250.txt  LON-00010-PT-02_S5.__90_120.tbl
LON-00010-PT-02_S5.__160_190.tbl  LON-00010-PT-02_S5.__190_220.txt  LON-00010-PT-02_S5.__60_90.tbl    LON-00010-PT-02_S5.__90_120.txt
LON-00010-PT-02_S5.__160_190.txt  LON-00010-PT-02_S5.__200_230.tbl  LON-00010-PT-02_S5.__60_90.txt
LON-00010-PT-02_S5.__170_200.tbl  LON-00010-PT-02_S5.__200_230.txt  LON-00010-PT-02_S5.__70_100.tbl
```

The command I used was:
`metabgc identify --sphmm_directory MetaBGC/Models/Cyclase_OxyN --prot_family_name Cyclase_OxyN --cohort_name OxyN --nucl_seq_directory ~/biobakery/kneaddata/final/test --seq_fmt FASTQ --pair_fmt split --r1_file_suffix repeats.removed.all.1.fastq --r2_file_suffix  repeats.removed.all.2.fastq --output_directory /home/amirza/MetaCBC --cpu 8`

I skipped the build step and used your models instead. Stool samples were sequenced by the Illumina NextSeq platform to generate shotgun metagenomic paired 2 × 150 bp reads. I couldn’t download the Toy dataset to test the program. I installed all the required software using the following codes on my linux system:

```
conda create --name MetaBGC python=3.6
conda install muscle=3.8.31=0
conda install -c bioconda hmmer=3.1b2 cd-hit blast emboss=6.5.7 
conda install -c r r-base=3.6.1
pip install metabgc
mkdir MetaCBC
cd MetaCBC
git clone https://github.com/donia-lab/MetaBGC.git
```

**Comment by abiswas-odu** [comment]
We have released version 2.0.0. Please try with it. I would suggest running the toy example. You have to click the link is a browser to download the file from google drive. The file is linked here: https://github.com/donia-lab/MetaBGC#perform-a-search

**Comment by aimirza** [comment]
These problems are resolved after running the new release. Thank you!
